components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/AbstractAPIManager.java

```diff
@@ -936,6 +936,25 @@ public void addSubscriber(Subscriber subscriber, String groupingId)
         apiMgtDAO.addSubscriber(subscriber, groupingId);
     }
 
+    public void addSubscriber(String username, String groupingId)
```

Comment:
Please note all the public method JavaDocs (which does not already have) are inherited from https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIManager.java class.